src/operator/tensor/init_op.h

```diff
@@ -644,6 +644,75 @@ inline bool RangeLikeShape(const nnvm::NodeAttrs& attrs,
   return true;
 }
 
+struct LogspaceParam : public dmlc::Parameter<LogspaceParam> {
```

Comment:
Create a `np_init_op.h` under `src/operator/numpy` folder and put all these code there.